src/ORM/Marshaller.php

```diff
@@ -167,6 +167,7 @@ public function one(array $data, array $options = [])
 
         $primaryKey = (array)$this->_table->primaryKey();
         $entityClass = $this->_table->entityClass();
+        /* @var Entity $entity */
```

Comment:
The code uses methods from EntityTrait. So Entity is the correct hint. I general find the EntityInterface kind of useless. It's to simple.